packet/mail.py

```diff
@@ -0,0 +1,19 @@
+from flask import render_template
+from flask_mail import Mail, Message
+
+from packet import app
+
+mail = Mail(app)
+
+
+def send_mail(packet):
+    if app.config['MAIL_PROD']:
+        recipients = ["<" + packet.freshman.rit_username + "@rit.edu>"]
+        msg = Message(subject="CSH Packet Starts " + packet.start.strftime('%D'),
```

Comment:
The `%D` directive isn't a very readable date in my opinion. Maybe try `"%A, %B %d"` which outputs `"Saturday, August 10"`? Since the year can safely be assumed that may be a more useful description of the date.